src/main/java/taxi/service/DriverService.java

```diff
@@ -3,4 +3,5 @@
 import taxi.model.Driver;
 
 public interface DriverService extends GenericService<Driver> {
+    Driver findByLogin(String login);
```

Comment:
```suggestion
    Optional<Driver> findByLogin(String login);
}
```

Reply:
In this case, I thought that returning the Optional at the service level would not be okay, so I did it this way, but at the DAO level I return the optional, so maybe we should wait for the mentors' comments?